test/lib/elements/Element.js

```diff
@@ -109,7 +109,8 @@ describe('Element', () => {
             let program = parseAndGetProgram('var first = 1; var second = 2;');
 
             expect(() => {
-                parseAndGetProgram('function foo(){ return\n; }').remove().remove();
+                program.remove();
```

Comment:
I just was surprised to see unrelated changes here, like in https://github.com/cst/cst/pull/88/files#r48984590